verilog/analysis/default_rules.h

```diff
@@ -66,6 +66,8 @@ constexpr const char* kDefaultRuleSet[] = {
     // TODO(b/117131903): "proper-parameter-declaration",
     // TODO(b/131637160): "signal-name-style",
     // TODO: "one-module-per-file",
+    // TODO(b/148911813): "constraint-name-style",
```

Comment:
This line should have already been removed recently.  Delete this line.

Reply:
done!